src/style.css

```diff
@@ -0,0 +1,102 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir.ttc);
+}
+
+/* #BIGregion */
+.big-form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 70px;
+  margin: 20px 0;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big-form:hover {
+  box-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+}
+
+.big-form__button {
+  margin: 0 17px 0 26px;
+  padding: 0;
+  border: none;
+  background-color: #fff;
+}
+
+.big-form__input {
+  width: 100%;
+  padding: 0;
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 22px;
+  color: #000;
+  border: none;
+}
+
+.big-form .big-form__input:focus {
+  outline: none;
+}
+
+/* #endregion */
+
+.small-form {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 42px;
+  margin-bottom: 20px;
+
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-sizing: border-box;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
```

Comment:
There are some common styles for `.big-form` and `.small-form`, use approach from checklist p2
![Screenshot 2023-06-22 at 15 15 10](https://github.com/mate-academy/layout_search-bar-airbnb/assets/32853034/7c9bd13f-8def-4ebf-8e29-8ac35540efc2)

![Screenshot 2023-06-22 at 15 15 03](https://github.com/mate-academy/layout_search-bar-airbnb/assets/32853034/dc17f9ae-3796-49f2-a949-5f9abfc01616)